fuzzers/qemu_systemmode/src/fuzzer_classic.rs

```diff
@@ -84,9 +88,23 @@ pub fn fuzz() {
 
     let mut run_client = |state: Option<_>, mut mgr, _core_id| {
         // Initialize QEMU
-        let args: Vec<String> = env::args().collect();
-        let env: Vec<(String, String)> = env::vars().collect();
-        let qemu = Qemu::init(&args, &env).unwrap();
+        let qemu = Qemu::builder()
+            .machine(qemu_config::Machine::new("mps2-an385"))
```

Comment:
Is there a benefit of having the `Machine` type over taking a string directly?

Reply:
In the first implem it was like that, but it led to a gigantic and ugly `Display` for `QemuConfig`.
Using an enum/struct for each option makes it possible to implement `Display` for each option and derive it for `QemuConfig`

Reply:
Internally we can still use a `Machine` but the exposed API can eat a string. I think this might "just work" if `Machine` implements `From<&str>`, or we need one extra flag in typed_builder